beta/session-1.md

```diff
@@ -0,0 +1,201 @@
+# Session 1: Setup and your first data versioning exercise
+
+Over the next hour, you will configure your computer to begin versioning a database in a container. 
+
+> For assistance, questions, or comments, please contact us via [Discord.](https://discord.gg/NVpqM7nNnT)
+
+> After completing the Session, please provide feedback [here.](https://forms.gle/6XSpqG7WrRsCNTrh7)
+
+# Install Titan, Liquibase, and Liquibase-Titan Wrapper 
+
+**IMPORTANT NOTES**
+
+- We currently do not support Docker on the Apple M1 Chip. 
+- Be aware that if you uninstall your current version of Docker to reinstall, the uninstall process will remove your images and volumes on your machine.
+
+## Step 1: Install Docker Desktop Community
+
+Download and install the latest version of [Docker Desktop Community.](https://www.docker.com/products/docker-desktop)
+
+> If you have previously installed Docker, verify that you are running 3.2.0 or higher. We have tested and require usage of Docker Desktop Community 3.2.0 or higher for this Beta. This includes the latest version, 3.6.0.
+
+**SYSTEM SPECIFIC SETUP**
+
+- On **Windows**, disable "Use the WSL2 Based Engine" in your Docker settings on Windows. We currently do not support Docker running via WSL2. You can always change this setting back later.
+
+![Image of Windows WSL2 Settings](images/docker_wsl2_settings.png)
+
+- On **Mac**, disable "Use gPRC FUSE for file sharing". 
```

Comment:
Should we also add the file sharing steps? This should prevent the issue you ran into.

Reply:
Absolutely. I added it to Session 3 where the diff stuff is. Don't want to clutter the first session any more.